br/pkg/utils/key.go

```diff
@@ -144,8 +145,8 @@ func ConvertBackupConfigKeyRange(startKey, endKey []byte, srcAPIVer, dstAPIVer k
 func EncodeKeyRange(start, end []byte) *KeyRange {
 	keyRange := KeyRange{}
 	keyRange.Start = codec.EncodeBytes(start)
-	if len(end) == 1 && end[0] == APIV2KeyPrefixEnd {
-		keyRange.End = []byte{APIV2KeyPrefixEnd}
+	if len(end) == APIV2KeyPrefixLen && bytes.Equal(end, APIV2KeyPrefixEnd[:]) {
```

Comment:
`if bytes.Equal(end, APIV2KeyPrefixEnd[:])` is enough ?

Reply:
yes, fixed. PTAL~